src/encoded/viewconfigs/auditview.py

```diff
@@ -248,8 +269,8 @@ def _summarize_no_audits(
         summary = []
         for xbucket in x_buckets:
             summary.append(counts.get(xbucket['key'], 0))
-        aggregations[group_by] = outer_bucket[group_by]['assay_title']
-        aggregations[group_by]['assay_title'] = summary
+        aggregations[group_by] = outer_bucket.get(group_by, {}).get(bucket_key, {})
```

Comment:
Good question. I think at a certain point, we cannot check everything in a defensive programming style. If aggression is null or aggression[group_by] is empty dict, that can be considered red-herring to a more serious problem rather than the problem.

Reply:
My concern is that we're opening up a new path in the code.  Previously the code would technically crash here if 'group_by' didn't exist in 'outer_bucket'.  Now we are allowing the code to function with ```aggregations[group_by] = {}```   

What happens when aggregations[group_by] is an empty dict?  We need to test this scenario if we're allowing it to happen now.

Reply:
I would carefully compare audit matrix and and experiment matrix code; I think only the matrices use group_by, and it's likely this code is somewhat redundant.